client/lib/olark-events/index.js

```diff
@@ -78,7 +78,7 @@ var OlarkEventEmitter = {
 	 */
 	olarkEventListener: function( event, ...eventArguments ) {
 		debug( 'Olark event: %s', event );
-		this.emit( event, eventArguments );
+		this.emit( event, ...eventArguments );
```

Comment:
This is a 0.5 out of 10 as far as being an important critique, but is `eventArguments` a little verbose when something like `args` seems like it would convey the same meaning?

```js
olarkApi( event, ( ...args ) => this.olarkEventListener( event, ...args ) );

…
olarkEventListener( event, ...args ) {
	this.emit( event, ...args );
}
```